src/makeRobot.js

```diff
@@ -38,7 +38,62 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    n: name,
+    wh: wheels,
+    vers: version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${this.n}, chip version: ${this.vers}, wheels: ${this.wh}`;
+    },
+
+    get location() {
+      return `${this.n}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goRight(value = 1) {
+      if (value > 0) {
+        this.coords.x += value;
+      };
+
+      return this;
+    },
+
+    goLeft(value = 1) {
+      if (value > 0) {
+        this.coords.x += -value;
```

Comment:
can we rewrite it like that?
```suggestion
        this.coords.x -= value;
```